src/cmd/troubleshoot/image.go

```diff
@@ -139,21 +139,12 @@ func checkComponentImagePullable(httpClient *http.Client, componentName string,
 			logWarningf("cannot pull image '%s' with version '%s' from registry '%s': %v",
 				componentImageInfo.image, componentImageInfo.version, registry, err)
 		} else {
-			logInfof("image '%s' with version '%s' exists on registry '%s",
+			logOkf("image '%s' with version '%s' exists on registry '%s",
 				componentImageInfo.image, componentImageInfo.version, registry)
-			imageWorks = true
 			break
 		}
 	}
 
-	if imageWorks {
-		logOkf("%s image '%s' found", componentName, componentImageInfo.registry+"/"+componentImageInfo.image)
-	} else {
-		// The image could not be pulled with any of the credentials
-		// Return as an error
-		return fmt.Errorf("%s image '%s' missing", componentName, componentImageInfo.registry+"/"+componentImageInfo.image)
```

Comment:
this error is no longer returned if the image can't be pulled

Reply:
readded